android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -372,8 +372,12 @@ public void run() {
 
   public void dismissSubKeysWindow() {
     try {
-      if (subKeysWindow != null && subKeysWindow.isShowing())
+      if (subKeysWindow != null && subKeysWindow.isShowing()) {
         subKeysWindow.dismiss();
+      }
+      if (subKeysList != null) {
+        subKeysList = null;
+      }
```

Comment:
```suggestion
      subKeysList = null;
```

Reply:
I also wonder why this is wrapped in an exception handler, although I know that isn't new -- what possible crash can we get from `subKeysWindow.isShowing()` or `subKeysWindow.dismiss()`?

Reply:
Hmm, I see it's always been in an exception handler from v2.8. Maybe it's just following the pattern from dissmissKeyPreview, though that block is inside a Runnable()

https://github.com/keymanapp/keyman/blob/2b3fa63cea49f47201226d74d35ad57f25d2385e/android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java#L758-L771